src/main/java/utils/Configs.java

```diff
@@ -0,0 +1,26 @@
+package utils;
+
+public interface Configs {
```

Comment:
I'm not sure it is a good idea. Constant interface is an anti-pattern. In the OOP paradigm interfaces should be implemented, hard to imagine a case where we can implement this interface with a lot of constants. Could you move it into enum or final class?

Reply:
I thought about it. I was considering instead using a final class with a private constructor to prevent it from being instantiated. Does it make sense?

Reply:
I changed to a final class with a private constructor